_data/docs.yml

```
@@ -46,6 +68,83 @@ apis:
+    ghlink: https://github.com/rapidsai/cucim
+    cllink: https://github.com/rapidsai/cucim/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
+  cuvs:
+    name: cuVS
+    path: cuvs
+    desc: 'cuVS is a library for GPU-accelerated vector search and clustering.'
+    ghlink: https://github.com/rapidsai/cuvs
+    cllink: https://github.com/rapidsai/cuvs/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
+  kvikio:
+    name: KvikIO
+    path: kvikio
+    desc: "KvikIO is a Python and C++ library for high performance file IO using GPUDirect Storage (GDS)."
+    ghlink: https://github.com/rapidsai/kvikio
+    cllink: https://github.com/rapidsai/kvikio/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
+  raft:
+    name: RAFT
+    path: raft
+    desc: "RAFT contains fundamental widely-used algorithms and primitives for vector search, machine learning, and information retrieval."
+    ghlink: https://github.com/rapidsai/raft
+    cllink: https://github.com/rapidsai/raft/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
+  dask-cuda:
+    name: Dask-CUDA
+    path: dask-cuda
+    desc: "Various utilities to improve deployment and management of Dask workers on CUDA-enabled systems."
+    ghlink: https://github.com/rapidsai/dask-cuda
+    cllink: https://github.com/rapidsai/dask-cuda/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
+  rmm:
+    name: RMM
+    path: rmm
+    desc: 'RAPIDS Memory Manager (RMM) is a central place for all device memory allocations in cuDF (C++ and Python) and other RAPIDS libraries. In addition, it is a replacement allocator for CUDA Device Memory (and CUDA Managed Memory) and a pool allocator to make CUDA device memory allocation / deallocation faster and asynchronous.'
+    ghlink: https://github.com/rapidsai/rmm
+    cllink: https://github.com/rapidsai/rmm/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
+  rapidsmpf:
+    name: RapidsMPF
+    path: rapidsmpf
+    desc: 'RAPIDS Multi-Process Foundation (rapidsmpf) is a collection of multi-GPU, distributed memory algorithms written in C++ and exposed to Python.'
+    ghlink: https://github.com/rapidsai/rapidsmpf
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 0
+      stable: 0
```

Pretty sure this can be 1 now. Maybe even legacy? Not sure if rapidsmpf was first published in 25.06 or 25.08.
```suggestion
      stable: 1
```

---

Sure, let's add this.

---

good way to test the `pre-commit` hook 😁 

https://github.com/rapidsai/docs/pull/669/commits/babe5c0e60797a748ca321e82ddb052bc104ca95